dev/tools/svg2dart/README.md

```diff
@@ -0,0 +1,5 @@
+# svg2dart
+
+A tool for converting SVG sequences as provided by our motion designers into a Dart representation.
+
+**Note: only a very limited set of SVG features is supported**
```

Comment:
We should describe in more detail what exactly is supported and what isn't.

It might be worth calling this something like "animated_icon_generator" rather than "svg2dart" since the current name is promising a lot more than it can do.

Reply:
done